lewis_emulators/keithley_2700/interfaces/stream_interface.py

```diff
@@ -10,68 +12,71 @@
-    in_terminator = "\r\n"
-    out_terminator = "\r\n"
+    in_terminator = "\r"
+    out_terminator = "\r"
     commands = {
-        CmdBuilder("get_idn").escape("*IDN?").build(),
-        CmdBuilder("empty_queue").escape(":SYST:CLE").build(),
-        CmdBuilder("clear_buffer").escape("TRAC:CLE").build(),
-        CmdBuilder("set_measurement").escape(":FUNC '").arg("VOLT:DC|VOLT:AC|CURR:DC|CURR:AC|RES|FRES|CONT|FREQ|PER")
-            .escape("'").build(),
-        CmdBuilder("get_measurement").escape(":FUNC?").build(),
-        CmdBuilder("set_buffer_feed").escape("TRAC:FEED ").arg("SENS|CALC|NONE").build(),
-        CmdBuilder("set_buffer_control").escape("TRAC:FEED:CONT ").arg("NEV|NEXT|ALW").build(),
-        CmdBuilder("set_buffer_state").escape("TRAC:CLE:AUTO ").arg("OFF|ON").build(),
-        CmdBuilder("get_buffer_state").escape("TRAC:CLE:AUTO?").build(),
-        CmdBuilder("get_next_buffer_location").escape("TRAC:NEXT?").build(),
-        CmdBuilder("get_buffer_stats").escape("TRAC:FREE?").build(),
-        CmdBuilder("get_readings_range",  arg_sep="").escape("TRAC:DATA:SEL? ").int().escape(",").int().build(),
-        CmdBuilder("set_buffer_size").escape("TRAC:POIN ").int().build(),
-        CmdBuilder("get_buffer_size").escape("TRAC:POIN?").build(),
-        CmdBuilder("set_time_stamp_format").escape("TRAC:TST:FORM ").arg("ABS|DELT").build(),
-        CmdBuilder("get_time_stamp_format").escape("TRAC:TST:FORM?").build(),
-        CmdBuilder("get_delay_state").escape("TRIG:DEL:AUTO?").build(),
-        CmdBuilder("set_delay_state").escape("TRIG:DEL:AUTO ").arg("OFF|ON").build(),
-        CmdBuilder("set_init_state").escape("INIT:CONT ").arg("OFF|ON").build(),
-        CmdBuilder("get_init_state").escape("INIT:CONT?").build(),
-        CmdBuilder("set_sample_count").escape("SAMP:COUN ").int().build(),
-        CmdBuilder("get_sample_count").escape("SAMP:COUN?").build(),
-        CmdBuilder("set_source").escape("TRIG:SOUR ").arg("IMM|TIM|MAN|BUS|EXT").build(),
-        CmdBuilder("set_data_elements").escape("FORM:ELEM READ, CHAN, TST").build(),
-        CmdBuilder("set_auto_range_status", arg_sep="").escape("FRES:RANG:AUTO ").arg("OFF|ON").build(),
-        CmdBuilder("get_auto_range_status").escape("FRES:RANG:AUTO?").build(),
-        CmdBuilder("set_resistance_digits", arg_sep="").escape(":FRES:DIG ").int().escape(", (@").
-        int().escape(":").int().escape(")").build(),
-        CmdBuilder("set_resistance_rate").escape(":FRES:NPLC ").float().build(),
-        CmdBuilder("set_scan_state").escape("ROUT:SCAN:LSEL ").arg("INT|NONE").build(),
-        CmdBuilder("get_scan_state").escape("ROUT:SCAN:LSEL").build(),
-        CmdBuilder("set_scan_channels", arg_sep="").escape("ROUT:SCAN (@").int().escape(":").int().escape(")").build(),
-        CmdBuilder("get_scan_channels").escape("ROUT:SCAN?").build()
+        # get_multicmds splits commands by ';' if multiple command strings are received
```

Comment:
Consider different formatting for these commands. I.e., 

````python
        CmdBuilder("set_measurement", ignore_case=True).escape(":FUNC '")
                                                       .arg("VOLT:DC|VOLT:AC|CURR:DC|CURR:AC|RES|FRES|CONT|FREQ|PER")
                                                       .escape("', (@").regex("[+-]?\d+").escape(":").regex("[+-]?\d+")
                                                       .escape(")").eos().build(),
````

Also, I believe the `ignore_case` argument is invalid.

Reply:
Thanks for the clarification.